### PR TITLE
Switch to Type B tiles

### PR DESCRIPTION
**Pull request by nkoenig**
All of the original Cave tiles have been renamed to include a "Type B" suffix. This suffix is meant to distinguish these tiles from the "Type A" cave tiles.

I used `sed` to do the replacements. @iche033 , do the *.dat files need to be regenerated?

Signed-off-by: Nate Koenig <nate@openrobotics.org>

### CODE REVIEW

**Comment by caguero** [comment]
If the dimensions and position of the tiles is the same, the `.dat` files can be reused.

**Comment by nkoenig** [comment]
Awesome, thanks. The tiles are exactly the same. Just the name changed.

File: subt_ign/worlds/simple_cave_01.tsv [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-				Cave Straight 01,180,0																		
-				Cave Straight 03,180,0																		
-				Cave Corner 02,-90,0	Cave Corner 01,90,0																	
-					Cave Straight 05,0,0,Phone@0 3.32 0.21 -93.96 0 0																	
-				Cave Corner 01,180,0	Cave 3 Way 01,0,0	Cave Corner 02,90,0																
-				Cave Straight 04,0,0		Cave Vertical Shaft,180,0																
-				Cave Corner 02,-90,0	Cave Vertical Shaft,-90,0,Backpack@5.11 0 0.03 0 0 -80	Cave 3 Way 01,90,4																
-						Cave Straight 02,0,4,Rescue Randy@0 -12.28 0 0 0 185																
-						Cave Cap,0,4																
+																Cave Corner 01,180,2	Cave Straight 02,90,2	Cave Cavern Split 02,0,2,Rescue Randy@3.29 8.14 -0.5 0 0 15	Cave Cavern Split 01,0,2 Type B,Cave Cap@0 37.5 25 0 0 180;Cave Cap@25 0 0 0 0 90
```
Comment (iche033) [inline subt_ign/worlds/simple_cave_01.tsv:1]: The `Type B` string is misplaced. It should come after the cave model name. I think this applies to all 3 simple cave tsv files.

Comment (nkoenig) [reply subt_ign/worlds/simple_cave_01.tsv]: Fixed in 86fa520e89cff2e6b63fe6ae4c3461eedfa23823

**Pull request merged**